dev/docker/iceberg-rest-server/rewrite_config.py

```diff
@@ -36,6 +36,9 @@
   "GRAVITINO_AZURE_TENANT_ID" : "azure-tenant-id",
   "GRAVITINO_AZURE_CLIENT_ID" : "azure-client-id",
   "GRAVITINO_AZURE_CLIENT_SECRET" : "azure-client-secret",
+  "GRAVITINO_OSS_ACCESS_KEY": "oss-access-key-id",
```

Comment:
Could you also add `GRAVITINO_OSS_REGION`, `GRAVITINO_OSS_ROLE_ARN`, `GRAVITINO_OSS_EXTERNAL_ID` to the mapping?